data-prepper-core/src/main/java/org/opensearch/dataprepper/peerforwarder/RemotePeerForwarder.java

```diff
@@ -222,6 +224,10 @@ private List<Record<Event>> populateBatchingQueue(final String destinationIp, fi
             }
         }
 
+        final int numberOfRecordsFailedToBatch = recordsFailedToBatch.size();
+        if (numberOfRecordsFailedToBatch > 0) {
+            LOG.warn("Failed to add {} records to the batching queue, processing locally.", numberOfRecordsFailedToBatch);
```

Comment:
Do we have an existing metric for this that we should increment?

Reply:
This rolls up into `recordsFailedToForward` here: https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-core/src/main/java/org/opensearch/dataprepper/peerforwarder/RemotePeerForwarder.java#L200